README.md

```diff
@@ -1,17 +1,13 @@
 # GOV.UK country and territory autocomplete - what it is and how to use it
 
----
-
-**This component is not currently maintained and we are unable to respond to issues and pull requests.**
+The country and territory autocomplete is a component that helps users choose countries and territories in answers. You should only use this component for services where users have to provide their country or territory.
 
-You can still raise issues or pull requests if you want to. As soon as we’re able to, we’ll prioritise dealing with security updates and any bugs that have been raised by people in the UK public sector. We cannot prioritise adding new features.
-
----
+For other types of service, you should use the accessible autocomplete. To find information about this component, go to the [accessible-autocomplete README](https://github.com/alphagov/accessible-autocomplete/blob/main/README.md).
 
 [![Build Status](https://travis-ci.com/alphagov/govuk-country-and-territory-autocomplete.svg?branch=main)](https://travis-ci.org/alphagov/govuk-country-and-territory-autocomplete)
 [![Greenkeeper badge](https://badges.greenkeeper.io/alphagov/govuk-country-and-territory-autocomplete.svg)](https://greenkeeper.io/)
```

Comment:
These can be removed (if they haven't been already), as they no longer work. You might need to rebase your PR?

Reply:
Thanks, will check and, if they're there, remove.